core/opendaq/reader/include/opendaq/block_reader.h

```diff
@@ -100,13 +107,15 @@ OPENDAQ_DECLARE_CLASS_FACTORY_WITH_INTERFACE(
     IBlockReader*, invalidatedReader,
     SampleType, valueReadType,
     SampleType, domainReadType,
-    SizeT, blockSize
+    SizeT, blockSize,
+    SizeT, overlap
```

Comment:
I acknowledge that we decided to not go for the builder pattern here, but I'm not 100% certain if we want to break the C API for readers at this point. 

It'd likely be better to keep the factories as they are, and use the builder pattern to add new construction parameters.

Reply:
Reverted